sdk/maps/azure-maps-search/README.md

```diff
@@ -26,47 +26,27 @@ pip install --pre azure-maps-search
 
 ### Authenticate the client
 
-In order to interact with the Document Translation feature service, you will need to create an instance of a client.
-An **endpoint** and **credential** are necessary to instantiate the client object.
+In order to interact with the Azure Maps Search service, you will need to create an instance of a client.
+An **client_id** and **credential** are necessary to instantiate the client object.
 
 ### Create the client with AzureKeyCredential
 
-To use an [subscription_key] as the `credential` parameter,
+Once completed, set the values of the client ID, and subscription key as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_SUBSCRIPTION_KEY`.
```

Comment:
Actually, subscription key doesn't need to pair with client ID. A new SearchClient constructor might be needed.